assets/js/modules/index.test.js

```diff
@@ -22,25 +22,37 @@
 import fs from 'fs';
 import path from 'path';
 
-const directories = ( relativePath ) => fs.readdirSync( path.join( __dirname, relativePath ) )
-	.filter( ( name ) => fs.lstatSync( path.join( __dirname, relativePath, name ) ).isDirectory() );
+function directories( relativePath ) {
+	const dir = path.join( __dirname, relativePath );
+	if ( ! fs.existsSync( dir ) ) {
+		return [];
+	}
 
-const getComponentNames = ( componentPath ) => fs.readdirSync( componentPath )
-	.filter( ( name ) => ! /^index|\.test\.js$/.test( name ) )
-	.map( ( name ) => name.replace( /\..*/, '' ) );
+	return fs.readdirSync( dir, { withFileTypes: true } )
+		.filter( ( file ) => file.isDirectory() )
+		.map( ( file ) => file.name );
+}
+
+function getComponentNames( componentPath ) {
+	return fs.readdirSync( componentPath )
+		.filter( ( name ) => ! /^index|\.test\.js$/.test( name ) )
+		.map( ( name ) => name.replace( /\..*/, '' ) );
+}
```

Comment:
Can you elaborate on why the changes here were necessary?

Reply:
The original test expects that all modules have the `components` folder and tries to check files and exports there. The new `analytics-4` module doesn't have components yet, so I have had to update this test to avoid false-positive fails.